src/app/_components/MainPageSections/DistrictList.js

```diff
@@ -7,16 +7,18 @@ import { CheckMark, Search } from '@icons';
 import { PillButton } from '@components/PillButton';
 import { Slide, Slider } from '@components/Slider';
 import { cn } from '@utils/cn';
+import { useBodyScrollLock } from '@/app/_hooks';
 
 const activeButtonStyles = 'pointer-events-none border-secondary-300 bg-secondary-300 font-semibold text-gray-900';
-const commonIconStyle = 'h-4 w-4 transition-all';
 
 export function DistrictList({ list, className }) {
   const [selected, setSelected] = useState(0);
   const handleClick = index => {
     setSelected(index);
   };
 
+  useBodyScrollLock(true, 'x');
+
```

Comment:
Isn't x axis always locked? I think we should have html {overflow-x: hidden;} styles globally

Reply:
Yeah, that makes a lot of sense. Changed it